Tests/PackageCollectionsTests/PackageCollectionsTests.swift

```diff
@@ -90,21 +116,14 @@ final class PackageCollectionsTests: XCTestCase {
             let list = try tsc_await { callback in packageCollections.listCollections(callback: callback) }
             XCTAssertEqual(list.count, mockCollections.count - 2, "list should be empty")
         }
-
-        do {
-            let unknownProfile = PackageCollectionsModel.Profile(name: "profile-\(UUID().uuidString)")
-            XCTAssertThrowsError(try tsc_await { callback in packageCollections.removeCollection(mockCollections[mockCollections.count - 2].source, from: unknownProfile, callback: callback) }, "expected error")
-        }
     }
 
-    func testDeleteFromStorageWhenLast() throws {
+    func testDeleteFromFromBothStorages() throws {
```

Comment:
```suggestion
    func testDeleteFromBothStorages() throws {
```

Reply:
will address in #3069 so we dont have to run CI again for one typo